process/css-styleguide/README.md

```diff
@@ -0,0 +1,168 @@
+# Meilisearch demos style guide
+
+Meilisearch Demos style guide contains a set of common UI elements along with their CSS for any developer to choose from to maintain consistent styling across the demos created for Meilisearch. If you are interested in creating a demo, you can refer to the common UI components and use those in your project.
+
+## About the style guides
+
+The style guide consists of a number of common UI components such as buttons, headings, search inputs that can be included while creating the UI of a project. It is based on the [Meilisearch Design System](https://www.figma.com/file/XXmUvsZzfZBKt1i2v12wf4/Design-System?node-id=1504%3A282).
+
+For creating beautiful in-brand demos, we encourage you to consult the [Meilisearch Guide to Design a Demo](https://www.figma.com/file/YUqMlOmApYd0D2aitPyDce/Guide-to-design-a-demo?node-id=2%3A8).
+
+## Components included
+
+- Button
+- Input
+- Logos
+- Modal
+- Search Input
+- Tag
+- Typography: headings and body text
+
+## Usage
+
+### Button
+
+1. The base class for styling a button is `.btn`.
+2. Change the color styling of the button by using the following classes:
+- `.btn-dodger-blue` 
+- `.btn-hot-pink`
+- `.btn-lila` 
+- `.btn-turquoise`
+- `.btn-watermelon`
+- `.btn-peppermint`
+- `.btn-candlelight`
+
+```html
+<button class="btn btn-dodger-blue" type="button">Button</button>
+```
+
+3. You can create a secondary button by adding the `btn-secondary` class
+
+```html
+<button class="btn btn-dodger-blue btn-secondary" type="button">Button</button>
+```
+
+4. To change the size of the button, you can add `btn-sm` or `btn-lg` to the class list.
+
+```html
+<button class="btn btn-sm btn-dodger-blue" type="button">Button</button>
+```
+
+
+Output :
+
+<img width="50%" alt="Capture d’écran 2022-08-23 à 10 33 08" src="https://user-images.githubusercontent.com/48251481/186461703-49f5d19b-128e-4b28-9348-f1838c69ab74.png">
+
+
+### Input
+
+1.  The parent class `.input` will assign some basic styling to the input.
+2.  Labels can also be styled depending upon various needs. The classes are `.input--label`, `.input--message`, `.input--error`, and `.input--message-error`
+
+```html
+<input
+  type="text"
+  class="input input--label"
+  placeholder="Write something"
+/>
+```
+
+Output :
+
+<img width="311" alt="Capture d’écran 2022-08-23 à 10 34 05" src="https://user-images.githubusercontent.com/48251481/186462336-24f8c55e-29fc-49c8-93c4-e45a90eb1e72.png">
+
+### Modal
+
+1. To render the content in a modal window, wrap it inside the parent element such as `div` and give it the `.modal` class.
```

Comment:
I don't really know what to think about this component. For me, this could also have been called `Card` as it isn't really a modal (no button to open a modal, no z-index or whatsoever...). It's just a style applied to a container 🤔 WDYT?

Reply:
I think we can even get rid of this component... It doesn't bring a lot of value

Reply:
Yes, totally agree 👍